modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -0,0 +1,216 @@
+As an alternative to using the `configure-ovs.sh` shell script to set a customized `br-ex` bridge on a bare-metal platform, create a `NodeNetworkConfigurationPolicy` custom resource (CR) that includes a customized `br-ex` bridge network configuration.
+
+:FeatureName: Creating a ``NodeNetworkConfigurationPolicy` CR that includes a customized `br-ex` bridge
+include::snippets/technology-preview.adoc[]
+
+This feature supports the following tasks:
+
+* Modifying the maximum transmission unit (MTU) for your cluster.
+* Modifying attributes of a different bond interface, such as MIImon (Media Independent Interface Monitor), bonding mode, or Quality of Service (QoS).
+* Updating DNS values.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+Consider the following use cases for creating manifest object that includes a customized `br-ex` bridge:
+
+* You want to make postinstallation changes to the bridge, such as changing the Open vSwitch (OVS) or OVN-Kubernetes `br-ex` bridge network. The `configure-ovs.sh` shell script does not support making postinstallation changes to the bridge.
+* You want to deploy the bridge on a different interface than the interface available on a host or server IP address.
+* You want to make advanced configurations to the bridge that are not possible with the `configure-ovs.sh` shell script. Using the script for these configurations might result in the bridge failing to connect multiple network interfaces and facilitating data forwarding between the interfaces.
+
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+[NOTE]
+====
+If you require an environment with a single network interface controller (NIC) and default network settings, use the `configure-ovs.sh` shell script.
+====
+
+After you install {op-system-first} and the system reboots, the Machine Config Operator injects Ignition configuration files into each node in your cluster, so that each node received the `br-ex` bridge network configuration. The Operator prevents `br-ex` bridge configuration conflicts by sending a signal to the `configure-ovs.sh` shell script to not configure the `br-ex` bridge.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+.Prerequisites
+
+* Optional: You have installed the link:https://nmstate.io/[`nmstate`] API so that you can validate the NMState configuration.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+.Procedure
+
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+. Create a NMState configuration file that has decoded base64 information for your customized `br-ex` bridge network. 
++
+.Example of an NMState configuration for a customized `br-ex` bridge network
+[source,yaml]
+----
+interfaces:
+- name: enp2s0 <1>
+  type: ethernet <2>
+  state: up <3>
+  ipv4:
+    enabled: false <4>
+  ipv6:
+    enabled: false
+- name: br-ex
+    type: ovs-bridge
```

Comment:
There are indentation errors in Line82-89, Line94, please help to update, thanks.

Reply:
:heavy_check_mark: